core/frontend/src/components/wizard/RequireInternet.vue

```diff
@@ -111,12 +111,12 @@ export default Vue.extend({
           if (!this.is_online) {
             this.timeout = setTimeout(() => {
               this.checkInternet()
-            }, 1000)
+            }, 5000)
           } else {
             clearInterval(this.timeout)
             this.timeout = setTimeout(() => {
               this.$emit('next')
-            }, 1000)
+            }, 5000)
```

Comment:
Maybe we can decrease this one to 2 seconds, since its when is already connected, just for user feedback.

Reply:
done!